<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisassemblerWidget</class>
 <widget class="QWidget" name="DisassemblerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color:gray</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WordSpinBox" name="startAddress">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:darkslategray</string>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="goToSelection">
       <property name="toolTip">
        <string>Go to PC address</string>
       </property>
       <property name="text">
        <string>☉</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="goToStart">
       <property name="toolTip">
        <string>Set PC to start address</string>
       </property>
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WordSpinBox</class>
   <extends>QSpinBox</extends>
   <header>wordspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
